templates/database/database.go

```diff
@@ -0,0 +1,69 @@
+package database
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/<% index .Params `database` %>"
+	"github.com/joho/godotenv"
+)
+
+var env map[string]string
+
+type Database struct {
```

Comment:
Is this just a hedge in case we want to add more stuff in the future?

Reply:
I made it thinking it would be easier for users to understand whats going on since it would be less lines on the `main.go`, I guess they'd have to read the Gorm docs if they want to do anything practical anyways

Reply:
Yeah.. Since you're using struct composition here, `Database` is actually just `gorm.db` with a different name.. 😉

Reply:
yeah i wanted to ship the TestConnection function with it to prove it actually connects but I guess that could be done in the main.go as well since its meant to be thrown away

Reply:
Wrapping some of the connect and disconnect stuff like you've done is fine. I don't mind leaving this as-is.